Heavy-Division/B78XH
# feat: GENX Texture and Animations

# Title



GENX Cowling, Spiral and Fanblades Texture updates


# Description 


Asobos Cowling, Fan Blades and Spiral were inaccurate for the GENX Engines, me and Ryanosaurus13#7000 have fixed this with new, updated textures.

# Source material


I used this image for reference: https://www.ainonline.com/sites/ainonline.com/files/uploads/2016/04/genxenginecopy.jpg

# Testing Instructions 


Download this PR and load into the sim, the Engine cowling, Fan blades and spirals should have updated textures, and should look like the image below.
# Test Results


![image](https://user-images.githubusercontent.com/80174397/214586829-ff430953-c2e5-4eb3-b9b3-3fc860506363.png)


# Discord Username
Toby 0#1557 & Ryanosaurus13#7000
